src/main/java/cc/polyfrost/oneconfig/config/annotations/HypixelKey.java

```diff
@@ -49,7 +49,10 @@
  * <p>
  *     Adding this annotation marks the field as non profile specific, see {@link NonProfileSpecific}.
  * </p>
+ *
+ * @deprecated Hypixel has drastically changed their API, and `/api new` no longer works. <a href="https://hypixel.net/threads/hypixel-developer-dashboard-public-api-changes-june-2023.5364455/">See Hypixel's API changes here.</a>
```

Comment:
didn't we look into providing an api proxy for mod devs? maybe link to ursa minor or smth

Reply:
Yea but we reach out to specific creators for it, not the other way around